platformio.ini

```diff
@@ -151,10 +151,10 @@ build_flags = -g
   -DCONFIG_LITTLEFS_FOR_IDF_3_2
 
 [scripts_defaults]
-extra_scripts             = pio-scripts/name-firmware.py
-                            pio-scripts/gzip-firmware.py
-                            pio-scripts/strip-floats.py
-                            pio-scripts/user_config_copy.py
+extra_scripts = pre:pio-scripts/set_version.py
+  post:pio-scripts/output_bins.py
+  post:pio-scripts/strip-floats.py
+  pre:pio-scripts/user_config_copy.py
```

Comment:
My OCD is saying would this be better formatted like...
```
extra_scripts = 
  pre:pio-scripts/set_version.py
  pre:pio-scripts/user_config_copy.py
  post:pio-scripts/output_bins.py
  post:pio-scripts/strip-floats.py
  ```

Reply:
I modified this part. But there are definitely many parts of the configs that could use some additional formatting and cleanup.